BlueprintUI/Sources/Internal/LayoutResultNode.swift

```diff
@@ -35,6 +35,8 @@ struct LayoutResultNode {
     var children: [(identifier: ElementIdentifier, node: LayoutResultNode)]
     
     init(element: Element, layoutAttributes: LayoutAttributes, content: ElementContent, environment: Environment) {
+        
+        element.assertIsValueType()
```

Comment:
This might be silly, but I've never thought to investigate until now; how expensive is a function call? Worth if-defing here too, since this is a hot path?

Reply:
Didn't profile, but google makes me think it's probably (hopefully?) optimized out

Reply:
So logging this even in debug with validations is fast enough that this is the result from logging the start and end dates is always extremely tiny, eg:

```
1.6239937394857407e-06
1.5599653124809265e-07
5.098991096019745e-08
1.2794043868780136e-07
4.807952791452408e-08
1.2293457984924316e-07
4.400499165058136e-08
6.402842700481415e-08
3.795139491558075e-08
3.8067810237407684e-08
3.702007234096527e-08
4.901085048913956e-08
3.993045538663864e-08
3.702007234096527e-08
3.702007234096527e-08
3.8067810237407684e-08
3.597233444452286e-08
4.7031790018081665e-08
5.098991096019745e-08
3.597233444452286e-08
3.899913281202316e-08
3.8067810237407684e-08
3.795139491558075e-08
```

Eg `0.000000037951`, aka like less than a ten millionth of a second... so I think we're good to go

Reply:
Nvm ignore this, ben provided a compile time way to do this